.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["es2015"], "react"],
```

Comment:
Do we care about pages with don't support ES6? If we delete es2015 we will have in code arrow functions, const, let etc.
@aju @vergilius  WDYF?